src/common-principles.md

```diff
@@ -259,7 +259,7 @@ field in `dataset_description.json` of each subdirectory of `derivatives` to:
 
 ```JSON
 {
-  "SourceDatasets": [ {"URL": "file://../../rawdata/"} ]
+  "SourceDatasets": [ {"URL": "../../rawdata/"} ]
```

Comment:
According to [4.3. URL writing](https://url.spec.whatwg.org/#url-writing):
> A **relative-URL string** must be one of the following, switching on [base URL](https://url.spec.whatwg.org/#concept-base-url)’s [scheme](https://url.spec.whatwg.org/#concept-url-scheme):
> * **A [special scheme](https://url.spec.whatwg.org/#special-scheme) that is not "`file`"**
>   * a [scheme-relative-special-URL string](https://url.spec.whatwg.org/#scheme-relative-special-url-string)
>   * a [path-absolute-URL string](https://url.spec.whatwg.org/#path-absolute-url-string)
>   * a [path-relative-scheme-less-URL string](https://url.spec.whatwg.org/#path-relative-scheme-less-url-string)
> * **"`file`"**
>   * a [scheme-relative-file-URL string](https://url.spec.whatwg.org/#scheme-relative-file-url-string)
>   * a [path-absolute-URL string](https://url.spec.whatwg.org/#path-absolute-url-string) if [base URL](https://url.spec.whatwg.org/#concept-base-url)’s [host](https://url.spec.whatwg.org/#concept-url-host) is an [empty host](https://url.spec.whatwg.org/#empty-host)
>   * a [path-absolute-non-Windows-file-URL string](https://url.spec.whatwg.org/#path-absolute-non-windows-file-url-string) if [base URL](https://url.spec.whatwg.org/#concept-base-url)’s [host](https://url.spec.whatwg.org/#concept-url-host) is not an [empty host](https://url.spec.whatwg.org/#empty-host)
>   * a [path-relative-scheme-less-URL string](https://url.spec.whatwg.org/#path-relative-scheme-less-url-string)
> * **Otherwise**
>   * a [scheme-relative-URL string](https://url.spec.whatwg.org/#scheme-relative-url-string)
>   * a [path-absolute-URL string](https://url.spec.whatwg.org/#path-absolute-url-string)
>   * a [path-relative-scheme-less-URL string](https://url.spec.whatwg.org/#path-relative-scheme-less-url-string)

Therefore, a relative URL cannot start with `file://`. Instead, a [**path-relative-scheme-less-URL**](https://url.spec.whatwg.org/#path-relative-scheme-less-url-string) like `../../rawdata/` is valid.

But then, I don't know if it works here...